core/bindings.rs

```diff
@@ -181,38 +180,15 @@ pub fn initialize_context<'s>(
   scope.escape(context)
 }
 
-pub unsafe fn slice_to_uint8array<'sc>(
-  deno_isolate: &mut Isolate,
+pub unsafe fn boxed_slice_to_uint8array<'sc>(
```

Comment:
I think the unsafe can be moved inside the function to only surround the `new_backing_store_from_boxed_slice`.

Reply:
ref https://github.com/denoland/rusty_v8/pull/247